apps.json

```
@@ -1051,11 +1055,11 @@
         "url": "https://github.com/YunoHost-Apps/elabftw_ynh"
     },
     "elasticsearch7": {
-        "category": "dev",
-        "state": "working",
         "antifeatures": [
             "not-totally-free"
         ],
+        "category": "dev",
+        "state": "working",
```

```suggestion
        "category": "dev",
        "level": 6,
        "state": "working",
```

as per https://ci-apps.yunohost.org/ci/job/12594